conf/flight_plans/tudelft/delft_bebop_convergence.xml

```diff
@@ -3,9 +3,9 @@
 <flight_plan alt="5" ground_alt="0" lat0="51 59 27.6" lon0="4 22 42.0" max_dist_from_home="200" name="Delft Basic" security_height="2">
   <header>
     #include "autopilot.h"
-    #include "subsystems/datalink/datalink.h"
-    #include "subsystems/electrical.h"
-    #include "subsystems/radio_control.h"
+    #include "modules/datalink/datalink.h"
```

Comment:
there is no problem to keep them, but all modules are visible from flight plan, so it is not even necessary to keep the manual includes (same for autopilot.h I think)